vendor/assets/javascripts/highcharts/themes/dark-unica.js

```diff
@@ -0,0 +1,213 @@
+/**
```

Comment:
As a actual answer, I'm going to keep them specifically because I'd rather not mess with the HighCharts release bundle. If we start micromanaging our plug-ins, we end up having to do a lot of manual culling and it increases the complexity of upgrades.